picard/script.py

```diff
@@ -1047,7 +1047,10 @@ def func_find(parser, haystack, needle):
     Returns:
         The zero-based index of the first occurrance of needle in haystack, or -1 if needle was not found.
```

Comment:
s/occurance/occurence/?

Reply:
The docstring needs to be updated to reflect the change